File: layout/src/main/java/com/baichang/android/develop/MainActivity.java
package com.baichang.android.develop;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import com.baichang.android.develop.bootomnavigation.BottomNavigationActivity;
import com.baichang.android.develop.coordinatorlayout.CoordinatorLayoutActivity;

public class MainActivity extends AppCompatActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
  }

  public void coordinatior(View view) {
    startActivity(new Intent(this, CoordinatorLayoutActivity.class));
  }

  public void bootomnavigation(View view) {
    startActivity(new Intent(this, BottomNavigationActivity.class));
  }
}

File: kotlin/src/main/java/com/example/kotlin/MainActivity.kt
package com.example.kotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    var quantity = 5
    val price: Double = 20.3
    val name: String = "大米"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        println("Hello Kotlin")
        Person("iCong").printName()

        println("单价:$price")
        println("数量:$quantity")
        println("产品：$name 总价:${quantity * price}")
    }
}
